webroot/styles/chat.css

```diff
@@ -102,6 +102,13 @@
   padding: .25rem;
 }
 
+/* Hide emoji button on small screens */
+@media screen and (max-width: 860px) {
```

Comment:
Would prefer if we did this in JS, but I think this should be good enough and easier for now.